### PR TITLE
Updated Billing

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated Billing with autobuilds parallel info.

### CODE REVIEW

File: docker-hub/billing/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Docker offers pricing plans that are tailored for individual developers and deve
 
 **For individuals:**
 
-The **Pro** plan includes unlimited public repositories, unlimited private repositories, unlimited [collaborators](../repos.md#collaborators-and-their-role) for public repositories, one [service account](../repos.md#service-accounts) for private repositories, and 2 parallel builds, starting at $5 per month with the annual subscription.
+The **Pro** plan includes unlimited public repositories, unlimited private repositories, unlimited [collaborators](../repos.md#collaborators-and-their-role) for public repositories, one [service account](../repos.md#service-accounts) for private repositories, and 5 parallel builds, starting at $5 per month with the annual subscription.
```
Comment (usha-mandya) [inline docker-hub/billing/index.md:18]: @jerae-duffin The parallel builds update looks good. Do you also want to add automated builds to the list of features highlighted in the Pro and Team plans?

Comment (jerae-duffin) [reply docker-hub/billing/index.md:18]: Agree @usha-mandya to include autobuilds here.

**Comment by jerae-duffin** [comment]
@shaunmulligan can you confirm which is correct? Second commit I included autobuilds in the Pro and Team plan.

File: docker-hub/billing/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Docker offers pricing plans that are tailored for individual developers and deve
 
 **For individuals:**
 
-The **Pro** plan includes unlimited public repositories, unlimited private repositories, unlimited [collaborators](../repos.md#collaborators-and-their-role) for public repositories, one [service account](../repos.md#service-accounts) for private repositories, and 2 parallel builds, starting at $5 per month with the annual subscription.
+The **Pro** plan includes unlimited public repositories, unlimited private repositories, unlimited [collaborators](../repos.md#collaborators-and-their-role) for public repositories, one [service account](../repos.md#service-accounts) for private repositories, autobuilds, and 5 parallel builds, starting at $5 per month with the annual subscription.
```
Comment (shaunmulligan) [inline docker-hub/billing/index.md:18]: perhaps we can reword it as ` ... , auto builds with 5 parallel builds, starting at ....`

File: docker-hub/billing/index.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ The **Free** plan includes unlimited public repositories and unlimited collabora
 
 **For development teams:**
 
-The **Team** plan includes unlimited public and unlimited private repositories starting at $25 per month for the first 5 users and $7 per month for each user thereafter with the annual subscription. In addition, the Team plan offers 3 parallel builds, advanced collaboration and management tools, including organization and team management with role-based access controls for the whole team.
+The **Team** plan includes unlimited public and unlimited private repositories starting at $25 per month for the first 5 users and $7 per month for each user thereafter with the annual subscription. In addition, the Team plan offers autobuilds, 15 parallel builds, advanced collaboration and management tools, including organization and team management with role-based access controls for the whole team.
```
Comment (shaunmulligan) [inline docker-hub/billing/index.md:28]: Same here as the above comment

**Review by shaunmulligan — COMMENTED** [review]
just one small rewording

File: docker-hub/billing/index.md (hunk 2 above)
Comment (jerae-duffin) [reply docker-hub/billing/index.md:18]: Thank you! @shaunmulligan Updated

**Review by usha-mandya — APPROVED** [review]
LGTM. Thanks @jerae-duffin

**Pull request merged**